main.nf

```diff
@@ -357,8 +407,10 @@ process dupradar {
                          expressionHist(DupMat=dm)
     title("Distribution of RPK values per gene")
     dev.off()
+    file.create("dup.done")
```

Comment:
Does this mean that MultiQC will fire as soon as dupRadar is finished? Can we make it dependent on featureCounts as well?